holoviews/operation/datashader.py

```diff
@@ -1793,36 +1793,17 @@ class inspect_points(Operation):
 
     hits = param.DataFrame(default=pd.DataFrame(), allow_None=True)
 
-    max_points = param.Integer(default=1, doc="""
+    max_hits = param.Integer(default=1, doc="""
        Maximum number of points to display within the mask of size
        pixels. Points are prioritized by distance from the cursor
        point. This means that the default value of one shows the single
-       closest sample to the cursor.""")
-
-    transformer = param.Callable(default=identity, doc="""
-       Function that can transform the hits dataframe before it is
-       passed to the hits_transformer and points_transformer for further
-       customization of the hits parameter and points output
-       respectively. Useful when no distinction is needed between the
-       hits_transformer and the points_transformer or when these later
-       transformers share the same initial code (e.g. common merges and
-       joins with other sources of data)""")
-
-    hits_transformer = param.Callable(default=identity, doc="""
-       Function that can transform the hits dataframe after applying the
-       transformer and before it is set on the hits parameter. When no
-       hits occur, the value passed to the callable is None. This can be
-       used to do multiple common tasks such as 1) subselecting
-       available columns 2) limiting the maximum number of hit rows
-       returned 3) performing joins with other data sources 4) returning
-       a dataframe with appropriate columns and dtypes when no hits
-       occur.""")
-
-    points_transformer = param.Callable(default=identity, doc="""
-      Function that transforms the hits dataframe after applying the
-      transformer and before it is passed to the Points. Can be used to
-      customize the value dimensions e.g. to implement custom hover
-      behavior.""")
+       closest sample to the cursor. Note that this limit is not applies
+       to the hits parameter.""")
```

Comment:
I suggested calling it `max_hits` when I thought it applied to the `hits` parameter; now that it's clarified I don't think `max_hits` is the right name.

Reply:
Sure. How about `max_elements` then?

Reply:
Sure.

Reply:
Addressed in 8811c71b9

Reply:
Seems confusing and overloaded with HoloViews elements. It's more like max_geoms.

Reply:
I thought it _was_ a limit on hv elements?  Geoms seems confusing to me. Maybe max_selected? max_matches?

Reply:
I vote for `max_indicators`

Reply:
Addressed in 420e5da97. @philippjfr told me he had no objection to `max_indicators`.